libpod/oci_conmon_common.go

```diff
@@ -429,14 +429,17 @@ func (r *ConmonOCIRuntime) StopContainer(ctr *Container, timeout uint, all bool)
 		}
 	}
 
-	if err := r.KillContainer(ctr, 9, all); err != nil {
+	if err := r.KillContainer(ctr, uint(unix.SIGKILL), all); err != nil {
 		// Again, check if the container is gone. If it is, exit cleanly.
 		err := unix.Kill(ctr.state.PID, 0)
-		if err == unix.ESRCH {
+		switch {
+		case err == nil:
+			return nil
```

Comment:
If the error is nil it means the the process is still alive and the kernel did not yet killed the process, no? Or maybe the pid got reassigned to a new process.

I don't think we can treat nil as no error, otherwise we can skip this check completely.

Reply:
Ouhla, you are right.  We are sending `0`, so `nil` is bad.